sample/src/main/java/com/theartofdev/edmodo/cropper/sample/CropImageViewOptions.java

```diff
@@ -42,4 +47,75 @@
   public boolean flipHorizontally;
 
   public boolean flipVertically;
+
+  public enum RatioType {
```

Comment:
I don't see added value in that

Reply:
yep, just added this for a more organized and readable code.
i thought that adding something like:
`} else if (mCropImageViewOptions.aspectRatio.first == 16
-              && mCropImageViewOptions.aspectRatio.second == 9) {
-            mCropImageViewOptions.aspectRatio = new Pair<>(9, 16);
-          }`
every-time a new option for ratio is added, would be messy compared to just adding to a switch block